app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+from app.players.player import Player
+
+
+class Elf(Player, ABC):
+    def __init__(self, nickname: str, musical_instrument: str):
+        super().__init__(nickname)
+        self._musical_instrument = musical_instrument
+
+    @abstractmethod
+    def play_elf_song(self):

```

Comment:
Method "play_elf_song" in Elf class is the same for both subclasses so you can use it here once